src/odemis/gui/model/__init__.py

```diff
@@ -1042,6 +1045,12 @@ def __init__(self, main):
         # For dichotomic mode
         self.dicho_seq = model.ListVA()  # list of 4 enumerated for each corner
 
+class EnzelAlignGUIData(ActuatorGUIData):
+    def __init__(self, main):
+        ActuatorGUIData.__init__(self, main)
+        self.viewLayout.value = VIEW_LAYOUT_VERTICAL
+        self.step_size = model.FloatVA(1e-6)
+        self.align_mode = StringEnumerated("Z alignment", choices=set(("Z alignment", "SEM alignment", "FLM alignment")))
```

Comment:
Be aware it's a little tricky to write the strings like this because it's easy to make a typo, and Python will not notice (until really late). So that's why in such case, I usually go for constants (eg `Z_ALIGN = "Z alignment"`). This way, a typo is detected early. For the SPARC2 alignment, that's not what I did though, I just kept the strings all lower case and without space, to reduce the chances of typo.